vscode_extension/src/test/suite/aacExecutableWrapper.test.ts

```diff
@@ -18,7 +11,24 @@ suite("AaC Executable Wrapper Test Suite", () => {
         };
 
         await aacWrapper.executeCommandWithArguments(versionCommand)
-            .then((output: string) => assert.ok(output.includes(configuration.getConfigurationItem("version"))))
-            .catch(reason => assert.fail(reason));
+            .then((output: string) => {
+                const searchString = "version: success";
+                assert.strictEqual(output.includes(searchString), true, `'${searchString}' not found in ${output}`);
+            });
+    });
+
+    test("it should execute the help command", async () => {
+        const helpDumpCommand = {
+            name: "help-dump",
+            description: "",
+            arguments: [],
+        };
```

Comment:
Wrote this initially to test "-h", but the test infrastructure isn't capable of executing "aac -h" at the moment.